src/main/java/com/epam/esm/model/Certificate.java

```diff
@@ -0,0 +1,80 @@
+package com.epam.esm.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.util.Set;
+
+public class Certificate extends Entity implements Serializable {
+
+   private String  name;
+   private String description;
+   private Double price;
```

Comment:
![image](https://user-images.githubusercontent.com/24493000/106357464-3f294900-6317-11eb-9769-97aadc2e461a.png)
https://stackoverflow.com/questions/1359817/using-bigdecimal-to-work-with-currencies